packages/xo-web/src/xo-app/vm/tab-network.js

```diff
@@ -331,6 +331,18 @@ const COLUMNS = [
     name: _('vifNetworkLabel'),
     sortCriteria: (vif, userData) => userData.networks[vif.$network].name_label,
   },
+  {
+    itemRenderer: vif => (
+      <pre>
+        <Text
+          value={vif.rateLimit || ''}
```

Comment:
Maybe `0` instead of an empty `string`
@julien-f what do you think about `rateLimit` initial value

Reply:
This makes it impossible to type `0` in the field, though.

Reply:
Fallback to `''` only for `undefined`.